eng/dashboardpack/dashboardpack.csproj

```diff
@@ -28,12 +28,16 @@
 
   <Target Name="BeforeBuild" BeforeTargets="Build">
     <MSBuild Projects="../../src/Aspire.Dashboard/Aspire.Dashboard.csproj" Targets="publish" Properties="Configuration=$(Configuration);Platform=$(Platform);TargetFramework=$(TargetFramework);RuntimeIdentifier=$(DashboardRuntime)" />
+
+    <!-- After publishing the project, we ensure that the published assets get packed in the nuspec. -->
+    <ItemGroup>
+      <None Include="$(DotNetOutputPath)Aspire.Dashboard/$(Configuration)/$(TargetFramework)/$(DashboardRuntime)/publish/**/*" Pack="true" PackagePath="tools/" />
```

Comment:
The wildcard can match nothing too. Instead, we should add that to a temporary item, and then to `None`. And error out if the item is empty.